### ISSUE TITLE
HParams for "T-DMCA" from Liu 2018 "Generating Wikipedia by Summarizing Long Sequences"

### ISSUE THREAD

**Issue by pltrdy** [tensorflow/tensor2tensor#886]
Hey,

I try to experiment with the Transformer-Decoder Memory Compressed Attention model from <https://arxiv.org/pdf/1801.10198.pdf>.

I'm wondering if the following HParams are relevant/enough to reproduce
```
@registry.register_hparams
def transformer_dmca():
    """Hyperparameters for T-DMCA of Liu et al., (2018)
    """
    hparams = transformer_moe_base()

    hparams.layer_types = "#locm/redm/locm/redm/locm"
    hparams.max_input_seq_length = 5000
    hparams.max_target_seq_length = 5000

    return hparams
```

Thanks.

**Comment by lukaszkaiser** [comment]
You're 100% right! Do you want to create a small PR with these fixes?

**Comment by pltrdy** [comment]
I'm currently experimenting with those HParams. I would definitely open a PR when I get evidence that's it's working as expected.
Thanks for your feedback.

**Comment by pltrdy** [comment]
@lukaszkaiser after experimenting a bit with it, i've some remarks:

1. thanks to tensorboard I've seen that the model does contain an encoder, which seems to have 5 layers, even if `hparams.num_encoder_layers = 0`, how 'normal' is it?
2. I'm having troubles with output lenght. At prediction time I may get empty output or highly repetitive, depending on `alpha` (value from 0.6 to 5), it seems really sensitive. 
3. have you got some interesting results on CNN/DM? I'm currently training a model.

**Comment by pltrdy** [comment]
@lzzk nope.

**Comment by senarvi** [comment]
Did you concatenate the inputs and targets yourself, or do you expect the model to do that?

**Comment by pltrdy** [comment]
@senarvi nope, but note that the model isn't really working, this could even be the solution

**Comment by pltrdy** [comment]
@lukaszkaiser is there something new about T-DMCA?
